arm/Microsoft.RecoveryServices/vaults/.parameters/parameters.json

```diff
@@ -243,6 +243,12 @@
                 }
             ]
         },
+        "backupStorageConfig":{
```

Comment:
Please update all parameter names you changed and manually run the **RecoveryServices: Vaults** in github actions to test the new module on your branch 
![image](https://user-images.githubusercontent.com/56914614/141275431-99d9e8d2-bace-4a92-bd38-e669b9f40b59.png)

You can also add a badge on the PR description to prove the correct run